src/knx-mqtt.js

```diff
@@ -93,11 +93,9 @@ let onKnxEvent = function (evt, dst, value, gad) {
     let isResponse = evt === 'GroupValue_Response';
     let mqttMessage = value;
     if (messageType === c.MESSAGE_TYPE_VALUE_ONLY) {
-        mqttMessage = !Buffer.isBuffer(value) ? "" + value : value
```

Comment:
We should do the same thing here right, and just return `value`?

Reply:
I'm not sure, because `JSON.stringify` only happens in the `MESSAGE_TYPE_FULL` branch.
Perhaps instead of `String(value)` it should be `JSON.stringify(value)`. I only see how the results would be different for `Buffer` and `undefined`, but it's the more correct thing to do.